test/integration/controller/jobs/pod/pod_controller_test.go

```diff
@@ -65,6 +68,8 @@ var _ = ginkgo.Describe("Pod controller", ginkgo.Ordered, ginkgo.ContinueOnFailu
 
 		ginkgo.BeforeAll(func() {
 			fwk.StartManager(ctx, cfg, managerSetup(
+				false,
+				false,
```

Comment:
Why don't we enable scheduler here if we did it in the past?

Reply:
Originally managerSetup function don't enable scheduler.